src/usb_moded-dbus.c

```diff
@@ -41,6 +42,25 @@
 static DBusConnection *dbus_connection_sys = NULL;
 extern gboolean rescue_mode;
 
+/**
+ * Issues "sig_usb_config_ind" signal.
+*/
+static void usb_moded_send_config_signal(const char *section, const char *key, const char *value)
+{
+  if (dbus_connection_sys)
+  {
+	DBusMessage* msg = dbus_message_new_signal(USB_MODE_OBJECT, USB_MODE_INTERFACE, USB_MODE_CONFIG_SIGNAL_NAME);
+	if (msg) {
+		dbus_message_append_args(msg, DBUS_TYPE_STRING, &section,
+		                              DBUS_TYPE_STRING, &key,
+		                              DBUS_TYPE_STRING, &value,
```

Comment:
Besides. Not only string arguments are easier to marshall and demarshall, they provide better compile-time type control on the consumer side (assuming that clients generate D-Bus stubs from the .xml file).